scripts/files/files_helper.py

```diff
@@ -1,9 +1,16 @@
 import os
+from enum import Enum
 from typing import Optional
 
 from scripts.gdal.gdalinfo import GdalInfo, gdal_info
 
 
+class ContentType(str, Enum):
+    TIFF = "image/tiff; application=geotiff; profile=cloud-optimized"
```

Comment:
this is a COG not a tiff, should we be more specific?

Reply:
https://docs.ogc.org/is/21-026/21-026.html#_media_types_for_any_data_encodings - From my understanding we should use `geotiff`? Or am I looking at the wrong place?